pkg/util/migrations/migrations.go

```diff
@@ -112,16 +119,15 @@ func VMIMigratableOnEviction(clusterConfig *virtconfig.ClusterConfig, vmi *v1.Vi
 }
 
 func ActiveMigrationExistsForVMI(migrationIndexer cache.Indexer, vmi *v1.VirtualMachineInstance) (bool, error) {
-	objs, err := migrationIndexer.ByIndex(cache.NamespaceIndex, vmi.Namespace)
+	objs, err := migrationIndexer.ByIndex(controller.ByVMINameIndex, fmt.Sprintf("%s/%s", vmi.Namespace, vmi.Name))
```

Comment:
Something to consider, for follow up, is to change the signature of the function